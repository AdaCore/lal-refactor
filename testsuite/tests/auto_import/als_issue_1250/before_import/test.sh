auto_import -P default.gpr --source main.adb --line 5 --column 15
