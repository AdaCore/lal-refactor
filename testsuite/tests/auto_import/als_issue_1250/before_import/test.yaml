description: >
    This test checks that, when the name is already qualified, only the with clause is added.

driver: shell_script
