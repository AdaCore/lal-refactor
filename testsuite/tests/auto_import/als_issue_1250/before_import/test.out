Available Imports:
Import: Test_1; Qualifier: 

Edits:
main.adb
1:1-1:1 with Test_1;


